File: debounce-plugin/src/main/java/com/sunyuan/click/debounce/utils/ClassUtil.java
package com.sunyuan.click.debounce.utils;

public class ClassUtil {

    public static boolean checkClassName(String className) {
        return className.endsWith(".class") && !className.equals("R.class") &&
                !className.startsWith("R$") && !className.equals("BuildConfig.class");
    }
}

File: debounce-plugin/src/main/java/com/sunyuan/click/debounce/utils/ClassUtil.kt
package com.sunyuan.click.debounce.utils

object ClassUtil {
    @JvmStatic
    fun checkClassName(className: String): Boolean {
        return className.endsWith(".class") && className != "R.class" &&
                !className.startsWith("R$") && className != "BuildConfig.class"
    }
}